src/models/TerraformComponentAttributeDefinition.js

```diff
@@ -0,0 +1,22 @@
+import { ComponentAttributeDefinition } from 'leto-modelizer-plugin-core';
+
+class TerraformComponentAttributeDefinition extends ComponentAttributeDefinition {
+  /**
+   * Override ComponentAttribute constructor with isDynamic property.
+   * @param {object} props - Terraform block properties.
+   * @param {boolean} [props.linkValue] - Value of the link.
```

Comment:
Rework on documentation because `linkValue` refer to the attribute name of the link in terraform value.
So update documentation and property name to linkAttribute.

Reply:
Done